sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -1860,12 +1879,19 @@ def test_sas_signed_identifier(self, tables_storage_account_name, tables_primary
 
             access_policy = AccessPolicy()
             access_policy.start = datetime(2011, 10, 11)
-            access_policy.expiry = datetime(2020, 10, 12)
+            access_policy.expiry = datetime(2025, 10, 12)
             access_policy.permission = TableSasPermissions(read=True)
             identifiers = {'testid': access_policy}
 
             self.table.set_table_access_policy(identifiers)
 
+            # token = self.get_sas(
+            #     tables_storage_account_name,
+            #     tables_primary_storage_account_key,
+            #     policy_id='testid'
+            # )
+            # self.scrubber.register_name_pair(token, "fake_token_value")
```

Comment:
This wasn't working?

Reply:
No, this is a separate issue where SAS tokens are stored in plain text in recording files. Opening a separate issue/PR for this